nss-day-cohort-51/nss-day-cohort-51.github.io
# mw-Logo-resize-168 uploaded the resized logo image as logo.svg; it's …

#168 
# Description - Updated logo image in above the fold/Hero section
uploaded the resized logo image as logo.svg. it's 6 KB; 
claimed victory over sluggish load time in 3G networks. thanks, Brenda!; 
Uploaded new logo192.png and logo512.png images. These two new images have the logo and the COHORT 51 with alien head that we use in our NavBar

## Type of change


# Testing Instructions

- Pull down branch and run website in browser using `npm start`
- Inspect the logo in the Description section/above the fold; the file should now be an SVG
- check that the file size 6KB 


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
